### PR TITLE
implemented AuthontificationService task

### PR DESCRIPTION
**Pull request by SaiyaJ1N**

### CODE REVIEW

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
+
+
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        if (userService.findByEmail(email) == null) {
+            return false;
+        }
+        if (email.equals(userService.findByEmail(email).getEmail()) && password.equals(userService.findByEmail(email).getPassword())) {
+            return true;
+        }
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/service/AuthenticationService.java:24]: use local variables
and try to remove redundant checks

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]
style check failed
please fix

File: src/main/java/mate/academy/service/AuthenticationService.java (hunk 1 above)
Comment (SaiyaJ1N) [reply src/main/java/mate/academy/service/AuthenticationService.java]: I fixed all failed checks, Shortened code using local variable and removed previous checks replacing them with return.

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 2]
```diff
@@ -1,16 +1,23 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
+
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/service/AuthenticationService.java:6]:
```suggestion
```

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request closed without merging**